packages/macro/test/js-t.ts

```diff
@@ -86,22 +86,19 @@ export default [
     name: "Support id and comment in t macro as callExpression",
     input: `
         import { t } from '@lingui/macro'
-        t({
-          id: 'msgId_2',
-          message: 'text',
-          comment: 'description for translators'
-        })
-        t({ id: 'msgId', comment: 'description for translators', message: plural(val, { one: '...', other: '...' }) })
+        const msg = t({ id: 'msgId', comment: 'description for translators', message: plural(val, { one: '...', other: '...' }) })
       `,
-    expected: `
-      import { i18n } from "@lingui/core"
-      /*i18n*/
-      i18n._({ id: "msgId_2", message: 'text', comment: 'description for translators' })
-      
+    expected: `import { i18n } from "@lingui/core";
+    const msg =
       /*i18n*/
-      i18n._({ id: "msgId", comment: 'description for translators', message: '{val, plural, one {...} other {...}}', values: {
-        val: val,
-      } })
+      {
```

Comment:
Actually, the descriptor must be wrapped in `i18n._` call. Otherwise the macro would be the same as `defineMessage`. I think the issue is that the `/* i18n */` comment must be *inside*:

```diff
- /*i18n*/i18n._({ id: "msgId_2", message: 'text', comment: 'description for translators' })
+ i18n._(/*i18n*/{ id: "msgId_2", message: 'text', comment: 'description for translators' })
```

Let me check how to fix it

Reply:
Yes, that was the issue.